api/migrations/0003_cognoma_genes.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.8 on 2016-11-14 00:24
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0002_alter_sample_fields'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Gene',
+            fields=[
+                ('entrezid', models.IntegerField(primary_key=True, serialize=False)),
```

Comment:
Should be `entrez_gene_id`. See [api docs](https://github.com/cognoma/core-service/blob/5ecbe388f7c368e0f0534adcced0824633fb4921/doc/api.md#gene-genes)